dot/core/service.go

```diff
@@ -497,6 +497,10 @@ func (s *Service) HandleSubmittedExtrinsic(ext types.Extrinsic) error {
 		return nil
 	}
 
+	if s.transactionState.IsInPool(ext) {
+		return nil
+	}
```

Comment:
we should also check if it's in the queue (i don't think that's already checked?)